src/screens/NewsScreen/Search.js

```diff
@@ -86,12 +87,12 @@ const styles = StyleSheet.create({
   wrapperIcon: {
     marginLeft: 8,
     alignSelf: 'center',
-    justifyContent: 'center',
+    justifyContent: 'center'
   },
   newPostText: {
-    color: COLORS.holyTosca,
+    color: COLORS.holytosca,
     marginRight: 11,
-    ...FONTS.h3,
+    ...FONTS.h3
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a change in the color constant from `COLORS.holyTosca` to `COLORS.holytosca`. Please ensure that the new color constant exists and is correct.



<!-- This is an auto-generated comment by OSS CodeRabbit -->